napari_allencell_annotator/view/viewer.py

```diff
@@ -44,4 +73,79 @@ def alert(self, alert_msg: str) -> None:
         show_info(alert_msg)
 
     def get_layers(self) -> List[Layer]:
+        """
+        Returns a list of all layers in the viewer.
+        """
         return list(self.viewer.layers)
+
+    def get_all_points_layers(self) -> List[Points]:
```

Comment:
this looks like a private method

Reply:
changed to private in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/ec029a71e9c03314bb78a974d9f09d4299abff5d